tests/postgres_tests/test_unaccent.py

```diff
@@ -42,6 +43,24 @@ def test_unaccent_chained(self):
             ordered=False
         )
 
+    def test_unaccent_chained_with_conforming_strings_off(self):
+        """
+        Chained unaccent causes the comparison string to be escaped.
```

Comment:
Docstrings should state the expected behavior and omit prefixes like "Make sure" since all tests make sure of things.